library/build.gradle

```diff
@@ -37,6 +37,10 @@ android {
     lintOptions {
         abortOnError false
     }
+
+    testOptions {
+        unitTests.returnDefaultValues = true
```

Comment:
Yup I know, but I didn't want to change too much the code.

The only way is to inject that ContentValues then... or a better idea?